spec/catalog/api/minion/approval_spec.rb

```diff
@@ -4,7 +4,10 @@
     let(:approval) do
       described_class.new("localhost", "9092")
     end
-    let(:message)         { ManageIQ::Messaging::ReceivedMessage.new(nil, event, payload, nil, nil, nil) }
+
+    let(:headers) { UserHeaderSpecHelper.default_headers }
```

Comment:
we could just use test values here {'a' => 1, 'b' => 2} we don't have to use the UserHeaderSpecHelper since it has no value we just need to make sure that whatever is in the message.header we can pass it along

Reply:
@mkanoor I agree. Why not just rip `UserHeaderSpecHelper` out and use a dummy test hash like you suggest.